src/lib/github/fetchGitHubRepos.ts

```diff
@@ -0,0 +1,39 @@
+import { getUserId } from '@/lib/auth/getUserId';
+import { getAccessToken } from '@/lib/auth/getAccessToken';
+import type { components } from '@octokit/openapi-types';
+
+export type GitHubRepo = components['schemas']['repository'];
+
+// TODO: Add pagination
+export async function fetchGitHubRepos(): Promise<GitHubRepo[]> {
+  const emptyResponse = Promise.resolve([] as GitHubRepo[]);
```

Comment:
This should work, given the function is marked as async?
```suggestion
  const emptyResponse = [] as GitHubRepo[];
```

Reply:
That's a good point. I switched to using a Result/Either pattern for the response type instead of hardcoding an empty response type like this.